server/parsec/components/postgresql/migrations/0006_tos.sql

```diff
@@ -0,0 +1,7 @@
+-- Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
+
+
```

Comment:
```suggestion
```

Reply:
I've modified the migration sql to stay consistent